cpp/models/abm/person.h

```diff
@@ -427,6 +419,20 @@ class Person
             loc, age, id);
     }
 
+    /**
+     * @brief Add TestResult to the Person
+     * @param[in] t The TimePoint of the test.
+     * @param[in] type The TestType of the test.
+     * @param[in] result The result of the test.
+    */
+    void add_test_result(TimePoint t, TestType type, bool result);
+
+    /**
+     * @brief Get the most recent TestResult performed from the Person based on the TestType.
+     * @param[in] type The TestType of the test.
```

Comment:
Perhaps we can add an `@return` here as in other function descriptions.

Reply:
I add a `@return` here as well as a comment regarding the` time_of_testing`.